.github/workflows/pr-github-checks.yml

```diff
@@ -10,14 +10,44 @@ on:
       - reopened
       - edited
       - synchronize
+      - labeled
```

Comment:
```suggestion
      - labeled
      - milestoned
```

Reply:
Could you please also copy the milestone check from the telemetry-manager repo?

Reply:
done